_includes/org_website_schema.html

```diff
@@ -0,0 +1 @@
+<!-- Structured Data markup starts --><script type="application/ld+json">{"@context":"http://schema.org","@type":"Organization","name":"PyConf Hyderabad 2017","url":"http://pyconf.hydpy.org/","logo":"http://pyconf.hydpy.org/img/sprites/logos.svg","sameAs":["https://www.facebook.com/HydPy/","https://github.com/HydPy/","https://twitter.com/hydPython"],"memberOf":{"@type":"Organization","name":"python software society of india"},"parentOrganization":"HydPy","areaServed":{"@type":"State","name":"Hyderabad, India"},"brand":{"@type":"Organization","name":"HydPy"},"email":"contact@hydpy.org","funder":[{"@type":"Organization","name":"Pramati"},{"@type":"Organization","name":"JetBrains"}],"sponsor":[{"@type":"Organization","name":"Pramati"},{"@type":"Organization","name":"JetBrains"}],"address":{"@type":"PostalAddress","name":"IIIT-Hyderabad","addressCountry":"India","addressLocality":"Hyderabad","addressRegion":"Telangana","postalCode":"500032","streetAddress":"Conference Hall","email":"contact@hydpy.org"}}</script><script type="application/ld+json">{"@context":"http://schema.org","@type":"WebSite","about":"PyConf Hyderabad is the regional gathering for the community that uses and develops the open-source Python Programming Language.","inLanguage":"English","publisher":["Hyderabad Python User Group","HydPy"],"potentialAction":{"@type":"SearchAction","target":"http://pyconf.hydpy.org/#s={search_term_string}","query-input":"required name=search_term_string"},"keywords":["python conference","python","python workshop","python hyderabad","pycon","pycon 2017","pyconf","october","workshop","hydpy","pyconf hyderabad","PyConfHyd","PyConf Hyderabad 2017","python conference hyderabad","python workshop hyderabad"],"copyrightHolder":[{"@type":"Organization","url":"http://pyconf.hydpy.org/","name":"PyConf Hyderabad"}],"name":"PyConf Hyderabad 2017","alternateName":"PyConfHyd 2017","url":"http://pyconf.hydpy.org/"}</script><div itemscope itemtype="http://www.schema.org/SiteNavigationElement"><span><a itemprop="url" href="http://pyconf.hydpy.org/"><span itemprop="name" content="Home"></span></a></span><span><a itemprop="url" href="http://pyconf.hydpy.org/schedule/"><span itemprop="name" content="Schedule"></span></a></span><span><a itemprop="url" href="http://pyconf.hydpy.org/speakers/"><span itemprop="name" content="Speakers"></span></a></span><span><a itemprop="url" href="https://pyconfhyd.talkfunnel.com/2017/"><span itemprop="name" content="Call for Proposal"></span></a></span><span><a itemprop="url" href="http://pyconf.hydpy.org/volunteers/"><span itemprop="name" content="Volunteers"></span></a></span><span><a itemprop="url" href="http://pyconf.hydpy.org/coc/"><span itemprop="name" content="Code of Conduct"></span></a></span></div><!-- Structured Data markup ends-->
```

Comment:
Can you write this in a structured manner instead using indentation so that it is readable similar to https://github.com/HydPy/hydpyconf2017/blob/master/_includes/schema-event.html . You don't have to worry about minification as it is taken care during build.